cli/app.go

```diff
@@ -1033,6 +1033,19 @@ var app = &cli.App{
 					},
 					Action: DataGetTrainingJob,
 				},
+				{
+					Name:      "logs",
+					Usage:     "gets training job logs from Viam cloud based on training job ID",
+					UsageText: createUsageText("train logs", []string{trainFlagJobID}, false),
+					Flags: []cli.Flag{
+						&cli.StringFlag{
+							Name:     trainFlagJobID,
+							Usage:    "training job ID",
+							Required: true,
+						},
+					},
+					Action: DataGetTrainingJobLogs,
```

Comment:
supernit and not your doing but I really think this should be an ML command, so `MLGetTrainingJobLogs`